cincinnati/src/plugins/internal/graph_builder/openshift_secondary_metadata_parser/plugin.rs

```diff
@@ -478,6 +491,52 @@ impl OpenshiftSecondaryMetadataParserPlugin {
         Ok(())
     }
 
+    async fn process_conditional_edges(
+        &self,
+        graph: &mut cincinnati::Graph,
+        data_dir: &PathBuf,
+    ) -> Fallible<()> {
+        let blocked_edges_dir = data_dir.join(BLOCKED_EDGES_DIR);
```

Comment:
nitpick: lets split out the deserialization part in a function and reuse it in `process_blocked_edges`?